components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/authz/AuthorizationHandlerManager.java

```diff
@@ -168,25 +165,15 @@ private OAuth2AuthorizeRespDTO validateAuthzRequest(OAuth2AuthorizeReqDTO authzR
 
         // Execute Internal SCOPE Validation.
         String[] authorizedInternalScopes = new String[0];
-        boolean isManagementApp = isManagementApp(authzReqDTO);
-        if (isManagementApp) {
-            if (log.isDebugEnabled()) {
-                log.debug("Handling the internal scope validation.");
-            }
-            JDBCPermissionBasedInternalScopeValidator scopeValidator = new JDBCPermissionBasedInternalScopeValidator();
-            authorizedInternalScopes = scopeValidator.validateScope(authzReqMsgCtx);
-            // Execute internal console scopes validation.
-            if (IdentityUtil.isSystemRolesEnabled()) {
-                RoleBasedInternalScopeValidator roleBasedInternalScopeValidator = new RoleBasedInternalScopeValidator();
-                String[] roleBasedInternalConsoleScopes = roleBasedInternalScopeValidator.validateScope(authzReqMsgCtx);
-                authorizedInternalScopes = (String[]) ArrayUtils
-                        .addAll(authorizedInternalScopes, roleBasedInternalConsoleScopes);
-            }
-        } else {
-            if (log.isDebugEnabled()) {
-                log.debug("Skipping the internal scope validation as the application is not" +
-                        " configured as Management App");
-            }
+        log.debug("Handling the internal scope validation.");
```

Comment:
Don't we need if (log.isDebugEnabled()) { ?

Reply:
No need, it is necessary to use isDebugEnabled when building the log message is costly. For example, concatenated strings, external method calls in logged message.